.github/workflows/release.yaml.disabled

```diff
@@ -0,0 +1,166 @@
+name: Java build EC2
```

Comment:
is this file still needed?

Reply:
No I will remove it.

tags are catched from the "push" event from the other pipeline